packages/grafana-ui/src/components/Table/TableCellInspector.tsx

```diff
@@ -28,8 +28,8 @@ export function TableCellInspector({ value, onDismiss, mode }: TableCellInspecto
     if (trimmedValue[0] === '{' || trimmedValue[0] === '[' || mode === 'code') {
       try {
         value = JSON.parse(value);
-        displayValue = JSON.stringify(value, null, '');
-      } catch {}
+        displayValue = JSON.stringify(value, null, ' ');
+      } catch { }
     }
   } else {
     displayValue = JSON.stringify(value, null, '');
```

Comment:
maybe let's indent with two spaces instead of one?

Reply:
actually maybe line 35 isnt meant to indent?

then we can probably just drop the trailing args `, null, ''`

Reply:
I think you're right about that one. I updated it 😄